### DISCUSSION TITLE
Is it possible to make the left and right calendars (selectRange) in the DatePicker move independently?

### DISCUSSION THREAD

**Discussion by pav2dva8** [Hacker0x01/react-datepicker#4159] [Q&A]
I have a date range picker, and the problem is when I change the month calendars are synchronized to show consecutive months.

`<DatePicker`
`          renderCustomHeader={(headerProps) => <DatePickerHeader {...headerProps} />}`
`         disabledKeyboardNavigation`
`          onChange={onChange}`
`          startDate={startDate}`
`          endDate={endDate}`
`          maxDate={new Date()}`
`          selectsRange`
`          inline`
`          monthsShown={2}`
`          calendarClassName="custom-datepicker"`
`          renderDayContents={renderDayContents}`
`        />`

Is there any way to separate them but keep selectRange logic?

eg left(Dec 2022) - right (Feb 2023). But if left month & year === right month & year, the right month should be left month + 1

**Comment by pav2dva8** [comment]
![Screenshot from 2023-07-13 19-47-01](https://github.com/Hacker0x01/react-datepicker/assets/70954883/2590af7a-8fc7-4899-8648-aa64d6ec989d)

**Comment by anjitpariyar** [comment]
I am too looking for something like this.For example, when selecting a date range that spans more than two months or even a year, it is necessary to have independent month and year dropdowns for each calendar. This allows users to easily navigate and select dates that are far apart without affecting the other calendar.
